src/textual/app.py

```diff
@@ -575,6 +576,12 @@ def __init__(
         self.set_class(self.dark, "-dark-mode")
         self.set_class(not self.dark, "-light-mode")
 
+        self.show_animations: AnimationsEnum = constants._get_textual_animations()
```

Comment:
Calling a method here, suggests it is not a constant. The `constants` module should really only expose values.

The name `show_animations` strongly suggests a bool, which is likely to confuse.

Reply:
Changed to `animations_level`.